server/manager/src/main/java/org/apache/accumulo/manager/tableOps/tableImport/ImportTable.java

```diff
@@ -53,8 +53,9 @@
 
 /**
  * Serialization updated for supporting multiple volumes in import table from 1L to 2L See:
- * https://github.com/apache/accumulo/issues/1053 https://github.com/apache/accumulo/pull/1060
- * https://github.com/apache/accumulo/issues/1576
+ * <a href="https://github.com/apache/accumulo/issues/1053">1053</a>
+ * <a href="https://github.com/apache/accumulo/pull/1060">1060</a>
+ * <a href="https://github.com/apache/accumulo/issues/1576">1576</a>
```

Comment:
None of these are needed. We don't need to embed references to historical changes to the code in the code itself. The git history does that already. This would get out of hand if we did this more often. These references can just be removed.

Reply:
I think I may have done this. I was trying to give context to the Version change since keeping track of serialized versions can be a pain. Maybe would be better to link with AccumloDataVersion or something else.